pkg/syncer/admissionhandler/validatepv.go

```diff
@@ -0,0 +1,80 @@
+package admissionhandler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/vsphere-csi-driver/v3/pkg/csi/service/logger"
+)
+
+func validatePv(ctx context.Context, req *admissionv1.AdmissionRequest) *admissionv1.AdmissionResponse {
```

Comment:
Does it invoke validatePV handler for Dynamically created PV as well?
It will be unnecessary to  validate dynamically created PV using webhook.